package.json

```
@@ -8,7 +8,8 @@
     "build": "tsc",
     "format": "prettier --write **/*.ts",
     "format-check": "prettier --check **/*.ts",
-    "test": "jest"
+    "test": "jest",
+    "pack": "ncc build"
```

```suggestion
    "build": "tsc && ncc build",
    "format": "prettier --write **/*.ts",
    "format-check": "prettier --check **/*.ts",
    "test": "jest"
 ```
 
 I'll remove the `pack` script and add the `ncc build` right after the `tsc`. That's because ncc uses the output of `tsc` to build the files. Having them in separate command increase the chance for user to forget running `npm run pack`.

---

Thanks for the suggestion! I also switched to a task-based approach to build the action.